src/collapse/guess.rs

```diff
@@ -111,64 +116,7 @@ impl Collapse for Folder {
         unreachable!()
     }
 
-    #[cfg(test)]
-    fn set_nstacks_per_job(&mut self, _: usize) {}
-
-    #[doc(hidden)]
     fn set_nthreads(&mut self, n: usize) {
         self.opt.nthreads = n;
     }
 }
-
-#[cfg(test)]
-mod tests {
-    use std::path::PathBuf;
-
-    use lazy_static::lazy_static;
-
-    use super::*;
-    use crate::collapse::tests_common;
-
-    lazy_static! {
```

Comment:
Where did these tests go?

Reply:
Now that not all the collapsers are multi-threaded, it doesn't make sense to have those tests for guess. (those tests only test whether the multi-threaded output is the same as the single-threaded output, if I'm remembering correctly).